### ISSUE TITLE
Run MP-CNN on torch-0.3.0.post4

### ISSUE THREAD

**Issue by hinhmd** [tuzhucheng/MP-CNN-Variants#4]
Training speed has increased significantly with the [new version of Pytorch](https://github.com/pytorch/pytorch/releases) (4x to 256x faster Softmax/LogSoftmax).
Please consider experimenting with the new version.

**Comment by tuzhucheng** [comment]
Yeah I'll update to PyTorch v0.3.0 and re-tune hyperparameters.

**Comment by tuzhucheng** [comment]
Upgraded to PyTorch 0.3.0, however, performance on some datasets dropped. Using #5 to keep track.
